### ISSUE TITLE
command not found: bunx

### ISSUE THREAD

**Issue by mkismy** [jdx/mise#1680]
**Describe the bug**
When install bun following the official installer `curl -fsSL https://bun.sh/install | bash`, `bunx` cli is also installed.
However when installing bun via mise, `bunx` is missing.

**To Reproduce**
```
$ mise use -g bun@latest
$ bunx
zsh: command not found: bunx
```

**Expected behavior**
`bunx` command executable.

**`mise doctor` output**

```text
mise version:
  2024.2.16 macos-arm64 (2024-02-15)

build:
  Target: aarch64-apple-darwin
  Features: DEFAULT, NATIVE_TLS
  Built: Thu, 15 Feb 2024 02:24:55 +0000
  Rust Version: rustc 1.75.0 (82e1608df 2023-12-21) (Homebrew)
  Profile: release

shell:
  /bin/zsh
  zsh 5.9 (x86_64-apple-darwin23.0)

mise dirs:
  data: /Users/user/.local/share/mise
  config: /Users/user/.config/mise
  cache: /Users/user/Library/Caches/mise
  state: /Users/user/.local/state/mise
  shims: /Users/user/.local/share/mise/shims

mise environment variables:
  MISE_SHELL=zsh

settings:
  activate_aggressive = false
  all_compile = false
  always_keep_download = false
  always_keep_install = false
  asdf_compat = false
  cargo_binstall = true
  color = true
  disable_default_shorthands = false
  disable_tools = []
  experimental = false
  go_default_packages_file = "~/.default-go-packages"
  go_download_mirror = "https://dl.google.com/go"
  go_repo = "https://github.com/golang/go"
  go_set_gopath = false
  go_set_goroot = true
  go_skip_checksum = false
  jobs = 4
  legacy_version_file = true
  legacy_version_file_disable_tools = []
  node_compile = false
  not_found_auto_install = true
  paranoid = false
  plugin_autoupdate_last_check_duration = "7d"
  python_compile = false
  python_default_packages_file = "/Users/user/.default-python-packages"
  python_pyenv_repo = "https://github.com/pyenv/pyenv.git"
  python_venv_auto_create = false
  raw = false
  trusted_config_paths = []
  quiet = false
  verbose = false
  yes = false
  ci = false
  debug = false
  trace = false
  log_level = "info"

  [status]
  missing_tools = "if_other_versions_installed"
  show_env = false
  show_tools = false


activated: yes
shims_on_path: no
config files:
  ~/.tool-versions
  ~/.config/mise/config.toml

plugins:
  bun      (core)
  deno     (core)
  go       (core)
  java     (core)
  node     (core)
  python   (core)
  ruby     (core)

toolset:
  python@3.11.7, bun@latest, node@20.10.0, deno@1.39.1

No problems found
```

**Additional context**

**Comment by kotahashihama** [comment]
same for me

**Comment by jdx** [comment]
I don't have much availability right now. Someone else should figure out what is going on here and recommend a fix.

**Comment by booniepepper** [comment]
Some context:
* `bunx` is "an alias for `bun x`" but also a CLI https://bun.sh/docs/cli/bunx
* `bunx` is not that new, looks like it was introduced at the end of 2022 https://twitter.com/jarredsumner/status/1606163655527059458
* Here's where we install `bun` https://github.com/jdx/mise/blob/bf8ff551d066b6a2c976687f6862cb7e2d50f231/src/plugins/core/bun.rs#L76-L91

Will throw something out for this

**Comment by booniepepper** [comment]
Before my change lands, you can fix this early with:

```shell
ln -s "$(which bun)" "$(dirname "$(which bun)")/bunx"
```

_(in a posix-y shell, with mise active and bun installed)_

**Comment by booniepepper** [comment]
The fix will be available in the next release, so `2024.2.17` or later.
